db/app.js

```diff
@@ -1,15 +1,31 @@
 const express = require('express')
 const app = express()
 
-const {getTopics, getAllEndpoints} = require('../db/controllers/nc-news-controller')
+const {getTopics, getAllEndpoints} = require('./controllers/topics-controller')
+const {getArticlesById} = require('./controllers/articles-controller')
 
 app.get('/api/topics', getTopics)
 app.get('/api', getAllEndpoints)
-
+app.get('/api/articles/:article_id', getArticlesById)
 app.all('/*', (req, res) => {
         res.status(404).send({mess: 'not found'})
     })
 app.use((err, req, res, next) => {
+    console.log(err)
+    if (err.status) {
+        res
+        .status([err.status])
+        .send({ mess: err.mess });
+    }
+    next(err);
+})
+app.use((err, req, res, next) => {
+    if (err.code === '22P02') {
+      res.status(400).send({mess: 'bad request'});
+    } else next(err)
+  });
+
```

Comment:
Your Error handlers are becoming quite big here - I think now is the time to pop them into a errors.controllers.js :)

Reply:
done